core/runtime/src/main/java/io/quarkiverse/mcp/server/runtime/EncoderResultMapper.java

```diff
@@ -0,0 +1,70 @@
+package io.quarkiverse.mcp.server.runtime;
+
+import java.util.List;
+
+import io.quarkiverse.mcp.server.Encoder;
+import io.quarkus.arc.All;
+import io.smallrye.mutiny.Uni;
+
+/**
+ *
+ * @param <ENCODED>
+ * @param <ENCODER>
+ * @param <RESPONSE>
+ */
```

Comment:
This isn't terribly helpful :)

Reply:
Yep, I should add some description here...